Samples/Swift/AppAttestExample/Podfile

```diff
@@ -1,10 +1,13 @@
+source 'https://github.com/CocoaPods/Specs.git'
+source 'https://github.com/firebase/SpecsDev.git'
+
 pod 'GoogleSignIn', :path => '../../../', :testspecs => ['unit']
 pod 'GoogleSignInSwiftSupport', :path => '../../../', :testspecs => ['unit']
 project 'AppAttestExample.xcodeproj'
 
 use_frameworks! :linkage => :static
 
 target 'AppAttestExample' do
-  pod 'AppCheckCore', :git => 'https://github.com/google/app-check.git', :tag => 'CocoaPods-0.1.0-alpha.1'
+  pod 'AppCheckCore'
```

Comment:
Note: there will likely be more breaking changes coming (could use 'CocoaPods-0.1.0-alpha.6' or leave it as is to be quickly notified when AppCheckCore changes)

Reply:
Thanks for the heads up there. I think I'll it this way since I like learning about breaking changes sooner rather than later. We're just merging into a feature branch anyway...